storage/blockchain/src/types.rs

```diff
@@ -143,6 +143,8 @@ pub struct PreRctOutputId {
     pub amount_index: AmountIndex,
 }
 
+impl Key for PreRctOutputId {}
+
```

Comment:
I think we'll need a multi-map abstraction for FCMPs btw (and it can replace this type).

This was the only type that needed to be implemented after removing the `Key` blanket impl.

Reply:
We might be able to use the same technique we currently use for pre-RCT outputs, I'll have to look into it more